Makefile

```
@@ -321,13 +321,11 @@ statusq-tests:
 	echo -e "\033[92mBuilding:\033[39m StatusQ Unit Tests"
 	cmake \
 		--build $(STATUSQ_BUILD_PATH) \
-                --target TestStatusQ \
 		$(HANDLE_OUTPUT)
 
 run-statusq-tests: statusq-tests
 	echo -e "\033[92mRunning:\033[39m StatusQ Unit Tests"
-	$(STATUSQ_BUILD_PATH)/bin/TestStatusQ
-
+	ctest -V --test-dir $(STATUSQ_BUILD_PATH)
```

```suggestion
	ctest -V --test-dir $(STATUSQ_BUILD_PATH) $(ARGS)
```

WDYT of this? It would allow us more control over the test without creating a new make target.
There are 2 cases where I think it would be useful:
1. Could use if for a flaky tests detection on CI:
E.g make -j10 run-statusq-tests ARGS="--output-junit ./testResults.xml --repeat-until-fail:10"
2. Allows us to easily run a specific test
E.g make -j10 run-statusq-tests ARGS="-R SubmodelProxyModelTest"

---

Definitely good idea!